traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -378,6 +379,8 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.1, http.MethodPut, `cdns/{id}/snapshot/?$`, crconfig.SnapshotHandler, auth.PrivLevelOperations, Authenticated, nil},
 		{1.1, http.MethodPut, `snapshot/{cdn}/?$`, crconfig.SnapshotHandler, auth.PrivLevelOperations, Authenticated, nil},
 
+		{1.1, http.MethodGet, `cdns/{cdn-name-or-id}/configfiles/ats/regex_revalidate.config/?(\.json)?$`, ats.GetRegexRevalidateDotConfig, auth.PrivLevelOperations, Authenticated, nil},
```

Comment:
Does this really need the `(\.json)?`? I tried the Perl version w/ a `.json` suffix, and I get a `400 Bad Request`.

Reply:
At some point, I'm planning to change the Go HTTP router to automatically route ".json" for all routes. Yes, they don't all officially support it, but it'll make routing much cleaner and easier, and reduce the chance of bugs, and we can remove it in API 2.0.

Until then, it will make that change more consistent if we start using `/?(\.json)?$` everywhere now.